source/val/validate_scopes.cpp

```diff
@@ -101,7 +101,7 @@ spv_result_t ValidateExecutionScope(ValidationState_t& _,
       if (spvOpcodeIsNonUniformGroupOperation(opcode) &&
           value != spv::Scope::Subgroup) {
         return _.diag(SPV_ERROR_INVALID_DATA, inst)
-               << _.VkErrorID(4642) << spvOpcodeString(opcode)
+               << _.VkErrorID(7951) << spvOpcodeString(opcode)
```

Comment:
This is not correct. The previous error code is still the right one. This message doesn't make sense for 7951.

Reply:
Sorry, you are right, I misread the new spec change

will revert this VU